pom.xml

```diff
@@ -181,6 +183,8 @@
                                         <argument>${cmake.osx_arch}</argument>
                                         <argument>-DS2N_NO_PQ_ASM=${cmake.s2nNoPqAsm}</argument>
                                         <argument>-DBUILD_TESTING=OFF</argument>
+                                        <argument>-DDISABLE_PERL=${cmake.disable_perl}</argument>
+                                        <argument>-DMY_ASSEMBLER_IS_TOO_OLD_FOR_512AVX=${cmake.disable_aws_lc_512avx}</argument>
```

Comment:
We took a different path in aws-crt-python:
https://github.com/awslabs/aws-crt-python/pull/468/files

instead of passing values in, we used our CMakeLists.txt to detect when it was an old GCC